src/main/java/com/spotinst/sdkjava/model/api/azure/statefulNode/ApiStatefulNodeDeallocationConfig.java

```diff
@@ -0,0 +1,111 @@
+package com.spotinst.sdkjava.model.api.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by Bansi Chapla on 23/11/2021.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiStatefulNodeDeallocationConfig implements IPartialUpdateEntity {
+
+    @JsonIgnore
+    private Set<String>                               isSet;
+    private Boolean                                   shouldTerminateVm;
+    private ApiStatefulNodeNetworkDeallocationConfig  networkDeallocationConfig;
```

Comment:
Do we need 4 different structures. They have same 2 parameters.

Reply:
I will make a note of this and make changes if needed